src/reporting/reportConversionResults.ts

```diff
@@ -93,3 +94,28 @@ const logMissingPlugins = (plugins: Set<string>, logger: Logger) => {
             .join(""),
     );
 };
+
+const logNotices = (converted: Map<string, ESLintRuleOptions>, logger: Logger) => {
+    const rulesWithNotices = Array.from(converted.values()).filter(
+        ruleOptions => ruleOptions.notices && ruleOptions.notices.length >= 1,
+    );
+
+    if (rulesWithNotices.length > 0) {
+        logger.stdout.write(chalk.yellowBright(`📢 ${rulesWithNotices.length} ESLint`));
+        logger.stdout.write(
+            chalk.yellowBright(rulesWithNotices.length == 1 ? ` rule behaves` : ` rules behave`),
+        );
+        logger.stdout.write(
+            chalk.yellowBright(` differently from their TSLint counterparts: 📢${EOL}`),
+        );
+
+        rulesWithNotices.forEach(rule => {
+            logger.stdout.write(chalk.yellow(`* ${rule.ruleName}:${EOL}`));
+            if (rule.notices !== undefined) {
+                rule.notices.forEach(notice => {
+                    logger.stdout.write(chalk.yellow(`- ${notice}${EOL}`));
```

Comment:
It'd be a little difficult to tell visual hierarchies in the output when the `*` has the same indentation as the `-`.
```suggestion
                    logger.stdout.write(chalk.yellow(`    - ${notice}${EOL}`));
```